### ISSUE TITLE
Callback ping

### ISSUE THREAD

**Issue by suntribe** [joomag/joomag-rest-php#1]
Maybe you could think about a callback ping? :) When user creates a new issue, the response will give him the issueTempID. Then, when the process will (un)successfully finish the conversion of the PDF, the API could ping some specific URL on the user's server with details (issueTempID, status of conversion, issue_id). That way, you save yourself servertime in a way that user doesn't need to ask the API every x minutes for the status of specific PDF... what do you think about such scenario?

**Comment by ghgugle** [comment]
Yes, we're thinking to create a webhook for conversion of the PDF. We will probably implement that in upcoming releases. But it would not affect the API.
